src/main/java/mate/academy/util/HashUtil.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.util;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+
+public class HashUtil {
+    private static final int SALT_SIZE = 16;
+    private static final String CRYPTO_ALGORITHM = "SHA-512";
+    
+    private HashUtil() {
+    }
+    
+    public static byte[] getSalt() {
+        SecureRandom secureRandom = new SecureRandom();

```

Comment:
minor 

you can make this a constant, so that you don't need to create a new instance each time